src/components/WaveformErrorBoundary.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { Alert } from 'react-bootstrap';
+
+class WaveformErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  state = {
+    showAlert: false
+  };
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true, showAlert: true };
+  }
+
+  handleDismiss = () => {
+    this.setState({ showAlert: false });
+  };
+
+  render() {
+    if (this.state.hasError && this.state.showAlert) {
+      return (
```

Comment:
@adamjarling Here `this.state.showAlert` comes up undefined when I run this.

Reply:
Ok, this is fixed up now.